test/worker_test.py

```diff
@@ -1322,6 +1322,17 @@ class NotImportedTask(luigi.Task):
             self.assertTrue(self.assistant.run())
             self.assertEqual(list(self.sch.task_list('DONE', '').keys()), [task.task_id])
 
+    def test_unimported_job_sends_failure_message(self):
+        class NotInAssistantTask(luigi.Task):
```

Comment:
You can make this more realistic by using the [`temporary_unloaded_module`](https://github.com/spotify/luigi/blob/6cf7091326af6eba64965767cef83f62a1bff53b/test/helpers.py#L193) helper.